internal/rotate.go

```diff
@@ -46,13 +50,27 @@ type CertRotationStep interface {
 
 // NewCertRotationHandler constructs a new handler to initiate a rotation with
 func NewCertRotationHandler(app *App, stateFile, estServer string) *CertRotationHandler {
+	corId := fmt.Sprintf("certs-%d", time.Now().Unix())
+	eventUrl := tomlGet(app.sota, "tls.server") + "/events"
+
+	target, err := LoadCurrentTarget(filepath.Join(tomlGet(app.sota, "storage.path"), "current-target"))
+	if err != nil {
+		log.Printf("Unable to parse current-target. Events posted to server will be missing content: %s", err)
```

Comment:
I think this message can mislead a user if `sota.toml` or `current-target` doesn't exist.

Reply:
that's why the message is there - if `current-target` doesn't exist. Its still better to send something than nothing, but we won't getting. Is there a better way to word this?

Reply:
I meant that in the case if `app.sota` does not exist we'll get `"Unable to parse current-target"`.
Or, in the case if `filepath.Join(tomlGet(app.sota, "storage.path"), "current-target`) is missing we'll get `"Unable to parse current-target"`.

In both cases the error message says that something wrong with the `current-target` content while the issue with absence of one of these two files.

Anyway, it's not important.

Reply:
yep - and its not possible in the code. you can't get to here with app.sota not existing. If `current-target` doesn't exist the error message will indicate that.